### PR TITLE
Further qobj tests

### PR DESCRIPTION
**Pull request by araza6**
Tests for https://github.com/qgrad/qgrad/pull/16.  I am writing more tests, so it is not ready for review yet

### CODE REVIEW

**Comment by quantshah** [comment]
Great, please rebase this with the master since I merged #16

**Comment by araza6** [comment]
I can't think of further tests for the quitp.rot method other than the one already added. Any pointers?

**Comment by araza6** [comment]
Hi guys,

This is about docstrings. For the `rot` method in `qgrad.qutip`, we have added docstring `numpy.array`[complex] for the return array. Since it is a `jax.numpy` array, should it be  `jax.numpy.array`[complex]?

**Comment by quantshah** [comment]
Yes that's right. But a normal numpy array also works so we should keep
that and explain in the docstring that for taking gradients one should use
the Jax numpy

**Comment by araza6** [comment]
@nathanshammah for setting a seed, and getting the fidelity of two same quantum states (your second point), isn't is the same as checking two same quantum states?

**Comment by araza6** [comment]
Closing this on https://github.com/qgrad/qgrad/pull/22
